S2S Integration Samples/README.md

```diff
@@ -35,4 +35,12 @@ S2SIntegration_ProcessPrintTask is used for
 - Receiving a webhook notification when the Task is created.
 - Getting a Print Job for the task and reading the job attributes.
 - Updating the job's "Color" attribute to "GrayScale".
-- Completing the Task.
\ No newline at end of file
+- Completing the Task.
+
+## Demos
+- Demo: [Create Printer]()
+- Demo: [Hold print job and implement task trigger]()
```

Comment:
Demo: Create `Task Triggers `on a printer and convert print jobs from color to grayscale